spec/prog/kubernetes/kubernetes_cluster_nexus_spec.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+require_relative "../../model/spec_helper"
+
+RSpec.describe Prog::Kubernetes::KubernetesClusterNexus do
+  subject(:nx) { described_class.new(Strand.new(id: "8148ebdf-66b8-8ed0-9c2f-8cfe93f5aa77")) }
```

Comment:
Is this hardcoded id needed?  I don't see it used anywhere else.  It appears to be for a `pg` UBID, which doesn't make much sense to me considering this is for k8s and not pg.